docs/config/01-configuration-file.md

```diff
@@ -280,6 +280,34 @@ httpsServerOptions: {
 **Description:** A list of log appenders to be used. See the documentation for [log4js] for more information.
 
 
+## middleware
+**Type:** Array
+
+**Default:** `[]`
+
+**Description:** List of names of additional middleware you want the karma server to use.
+
+You must have installed the middleware via a plugin/framework (either inline or via NPM). An example of custom inline middleware is shown below. Additional information can be found in [plugins].
```

Comment:
It might be good to mention that this is express/connect middleware, and have a link to their docs on how to write one.